### ISSUE TITLE
Add startup command for ssh in config

### ISSUE THREAD

**Issue by axelburks** [blinksh/blink#575]
Could you please add an option for ssh in config to add startup command, not only for mosh.
I know that I could start ssh with command in shell command, but the command I want to run on the server usually is too long or complicated to type it. 
Please thinking about that. Thanks.

**Comment by billybooth** [comment]
Yes, I can't see how this is available for mosh but not ssh...  I prefer to set up designated connections with startup commands for complicated commands that I need to execute regularly or somewhat regularly on specific servers...

**Comment by HeadCase** [comment]
Plus one for this. I assumed it was an oversight of UI design that the startup command was tucked in under MOSH, only to find myself astonished to discover that it apparently is only available for MOSH.
